Adyen.docc/Contributing.md

```diff
@@ -1,4 +1,5 @@
 # Contributing
+#
```

Comment:
This empty line is moving the text from the header to the body which supresses the superfluous `Summary` heading